src/Command/DbDump.php

```diff
@@ -74,22 +94,37 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $helper = $this->getHelper('question');
-        $question = new ConfirmationQuestion(
-            'We suggest to enable maintenance mode before running this command. Do you want to continue [y/N]?',
-            false
-        );
-        if (!$helper->ask($input, $output, $question) && $input->isInteractive()) {
-            return null;
+        if ($input->isInteractive()) {
+            $helper = $this->getHelper('question');
+
+            $questionParts = [
+                'The db-dump command requires the site to be placed into maintenance mode. ',
```

Comment:
@meker12 Can you review this text?  Seems like it could be more brief...
"This command will place the site into maintenance mode during execution.  During this time, your site will not receive traffic (Please see dev docs for more information).  Do you wish to proceed? (y/N)?"